pkgs/top-level/python-packages.nix

```diff
@@ -6252,6 +6254,10 @@ in {
 
   paperwork-backend = callPackage ../applications/office/paperwork/backend.nix { };
 
+  papis = callPackage ../applications/office/papis { };
```

Comment:
This is not correct. If its a Python library then it should be called from here but the expression should be in `pkgs/development/python-modules`

Reply:
But it is not a library?

Reply:
Then you should also add an entry to `all-packages.nix` i.e, `papis = python3.pkgs.papis`

Reply:
Then it should have an entry in `all-packages.nix`, but it should *not* be called via `python-packages.nix`.

Reply:
Moved to `all-packages.nix`.